calico/felix/actor.py

```diff
@@ -385,7 +426,8 @@ class Message(object):
     """
     Message passed to an actor.
     """
-    def __init__(self, msg_id,  method, results, caller_path, recipient,
+
+    def __init__(self, msg_id, method, results, caller_path, recipient,
```

Comment:
You've added more blank lines here (although good catch on the extra space). Any reason for that?